app/views/apis/detail.html

```diff
@@ -202,7 +202,7 @@
                 </div>
 
                 <iframe iframe-auto-size
-                        local-iframe="{{getTrustedUrl(settings.currentPath + 'swagger-console.html?url=' + api.url + '&host=' + api.swaggerPreferences.host + '&basePath=' + api.swaggerPreferences.basePath)}}">
```

Comment:
If the local-iframe is not used anymore, please delete it from the directive.js too.